go/oasis-test-runner/oasis/oasis.go

```diff
@@ -245,6 +253,12 @@ type NetworkCfg struct { // nolint: maligned
 	// A set of log watcher handler factories used by default on all nodes
 	// created in this test network.
 	DefaultLogWatcherHandlerFactories []log.WatcherHandlerFactory `json:"-"`
+
+	// UseCustomGrpcSocketPath specifies whether nodes should use internal.sock in datadir or externally-provided.
+	//
+	// WARNING: Using arbitrary unix socket paths poses security risk! This flag should only be used for debugging and
+	// end-to-end tests!
+	UseCustomGrpcSocketPath bool `json:"-"`
```

Comment:
Maybe consider renaming it to `UseShortGrpcSocketPaths` to better specify what it does. Because using this option does not allow one to specify a custom path, it just makes the test runner generate short paths and use them. Then also rename all the other related variables.

Reply:
This was not changed?

Reply:
Oh, I see you only added the "s" at the end, but in my proposal Custom should be replaced with Short.